pkg/helmclient/install.go

```diff
@@ -67,4 +67,5 @@ func (options InstallOptions) configure(action *action.Install, namespace string
 	action.ReleaseName = options.ReleaseName
 	action.Timeout = options.Timeout
 	action.Wait = options.Wait
+	action.SkipCRDs = options.SkipCRDs
```

Comment:
Nice but the flag also needs to be set in update.go.

Reply:
Not sure why? Since we don't use install flag (`-i`), there isn't a need for skip CRDS.

Reply:
Right, good catch. 👍